### DISCUSSION TITLE
Option to disable the KO black screen

### DISCUSSION THREAD

**Discussion by lIllIIlI** [FakeFishGames/Barotrauma#14564] [Ideas and suggestions]
I do not know why it was imposed forcefully upon every player, because most don't play with traitors enabled and dont have to bother with the griefers.
It does not make for a fun video when your screen just turns black and you cannot see how the rest of your crew gets slaughtered; all you can do now is just tab out or play something else while your crew members spends the next 20-60 minutes bringing you back to life. (Neurotrauma moment)
It simply is not fun, at all. 
I suggest leaving it as a toggle in the server/campaign settings.

**Comment by lIllIIlI** [comment]
https://www.reddit.com/r/Barotrauma/comments/iubngb/screens_should_be_blacked_out_while_unconscious/
The most sane comment at the bottom basically highlighted the problems in all of my campaigns. If there is no stimulation, the player gets bored. It is simply not fun for people to stare at a black screen.
It's fine when it's for a few seconds or a minute, but for 10 minutes? an hour? 

There was also this ticket here https://github.com/FakeFishGames/Barotrauma/discussions/12360, the author specifically mentioned that it should be a toggle in the server settings, yet I see no such option still in 2024.

**Comment by Regalis11** [comment]
I'm not sure what you mean. There's no black screen when you're knocked out - are you using some kind of a mod that introduces this? Neurotrauma perhaps?

**Comment by UnhappyWithMyUsername** [comment]
If you're playing with Hungry Europans, then that's where it's coming from. Putting another stun override mod above it in your load order will get rid of the black screen.
Other people suggested to use Stun Visual Effects ( https://steamcommunity.com/sharedfiles/filedetails/?id=3047651773 ) to remove Hungry Europans black screen.

**Comment by Regalis11** [comment]
Closing, since this seems to be an issue caused by mods, not something we can address. I hope the suggestions by UnhappyWithMyUsername above help you resolve the issue!
